client/tests/register_test.py

```diff
@@ -90,6 +90,37 @@ def my_function(df):
     # Checks that Transformation definition does not error when converting to source
     dec.to_source()
 
+@pytest.mark.parametrize(
+    "resource_serving_client_fxt,is_local",
+    [
+        ('resource_serving_clients', True),
+        ('resource_serving_clients', False),
+    ]
+)
+def test_valid_model_registration(resource_serving_client_fxt, is_local, request):
+    resource_client = request.getfixturevalue(resource_serving_client_fxt)(is_local)[0];
```

Comment:
This is getting the fixture. I add the `_fxt` suffix to denote that the argument is really the name of the fixture rather than the fixture itself. I can change this if it's throwing anyone off.

Reply:
Ah I see